### ISSUE TITLE
Speaking track info when navigating to new tracks.

### ISSUE THREAD

**Issue by monkeypusher69** [chigkim/Flogic#8]
Now that logic seems to seek tracks when navigating to them with up and down arrows, is it possible to add the functionality with Flogic to speak info relative to the currently selected track. For example, if the track is Soloed, Muted, how many regions are on the track, if the track is armed for recording or if input monitoring is on, pan value, etc etc. I am thinking it would only need to speak things like solo, mute, arm or input monitoring if they are on but say nothing about  their state if they are off. Don't say anything if its a empty track but speak number of regions if there is regions on that track. Not sure what if any of this could be implemented, but Some or all this info could be useful.

**Comment by chigkim** [comment]
Is there a particular reason why you want this information consolidated?
Every information you mentioned has its own dedicated shortcut.

**Comment by monkeypusher69** [comment]
Is there currently a way to query number of regions on a track? I mostly suggested for ease of use while navigating the track headers area without having t to press multiple keys. Also the region info could be useful for “confirmation” of sorts for some editing task.

**Comment by chigkim** [comment]
If you have inspector on, pressing r will tell you how many regions are 
selected. If you use temporary inspector, accent+r will do the same.
If you press up or down arrow to select a track, all the regions on that 
track will be selected. Thus you can find out how many regions are on a 
track by selecting a track with up/down and pressing r.
If you press right or left arrow to select a region, pressing r will 
tell you the name of the region.

**Comment by monkeypusher69** [comment]
My mistake. I didn’t realize R was speaking the info that was available next to the region inspector table. The times I guess I ended up using it  and based on my initial understanding of the read me I thought it only  spoke the selected region. Not sure how I never realized it was echoing the info from the region inspector. I have also noticed recently that when you initially interact with the track header area it speaks the number of regions on a track.   I still maintain that it could be useful info to have echoed while navigating the tracks header area but I take your point regarding the availability of other shortcuts to acquire that info.
